Kuba krj hmwk 4

kuba-krj opened the pull request.

Commit:
Ad hmw1

kuba-krj committed:
Add files via upload

pbiecek commented:
wrong folder (see other solutions)
lack of task 1 (here some math is needed)
lack of plots for task 2 (here a html file is expected)

Commit:
Move to corect directory

Commit:
Add plotly plots and html file

Commit:
Correct notebook

Commit:
Remove csv

kuba-krj commented:
I have changed folder, uploaded the `plotly` images that were not visible and added mathematical calculation to Task 1 instead of Monte Carlo estimation. Sorry for all of mistakes in the primary solution.

pbiecek commented:
50% - very limited comments related to task 2
